includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-abstract.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
```
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 19 | ERROR | Missing @throws tag in function comment
 92 | ERROR | 'string' type declaration is not present in PHP version 5.6 or earlier
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Reply:
fixed except the one related to prior to PHP 5.6.

Reply:
fixed/ignored accordingly :D